pkg/pulumiyaml/codegen/eject_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2025, Pulumi Corporation.  All rights reserved.
+
+package codegen
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_loadProjectFromDir(t *testing.T) {
+	t.Parallel()
+	testPath, err := filepath.Abs("./testdata/simple_project")
```

Comment:
Maybe use `filepath.Join` here to assemble the path?  Same in the other tests below.